doc/ch-image.rst

```diff
@@ -2106,6 +2106,195 @@ in the remote registry, so we don’t upload it again.)
 Delete all images and cache from ch-image builder storage.
 
 
+:code:`shell`
```

Comment:
So this proposes calling it `ch-image shell` rather than `ch-image modify`. I’m open to argument but this seemed more specific.